services/src/main/java/org/jboss/windup/web/services/rest/AnalysisContextEndpoint.java

```diff
@@ -25,6 +27,11 @@
     @Path("{id}")
     AnalysisContext get(@PathParam("id") Long id);
 
+
+    @PUT
+    @Path("storeDefaultConfigForProject/{projectId}")
+    AnalysisContext storeDefaultConfigForProject(@Valid AnalysisContext analysisContext, @PathParam("projectId") @NotNull Long projectId);
+
```

Comment:
+1 - I don't understand the difference between these two use-cases. I think that the approach David is suggesting here is simpler and more robust as well.